### DISCUSSION TITLE
Gboard Glide typing (Swipe) not working?

### DISCUSSION THREAD

**Discussion by ekadm** [AppGround-io/bluetooth-keyboard-and-mouse-support#272] [Bugs & Problems]
Hello, just got the app but glide typing doesnt work for me. I saw on thee reviews that it does work for some. I have the paid version on my pixel 5, looking to glide type on lenovo chromebook duet 5. Any ideas? Thanks!

**Comment by parkerlreed** [comment]
You probably have to switch to the text input box and not the real time typing.

So select box on phone, swipe out words, send input
